docs/References/troubleshooting.md

```diff
@@ -57,3 +57,60 @@ If you are unable to access your cluster via kubectl from the dashboard, check o
+
+It also records logs for audit and compliance, so you can see who and when did what on your K8s infrastructure. Paralus can be used with a web GUI, CLI, or API.
+
+### 2. Is Paralus a CNCF Project?
+
+Paralus is a [CNCF Sandbox project](https://www.cncf.io/projects/paralus/).
+
+### 3. How and where can I try it myself?
+
+You can download and install Paralus for free on [GitHub](https://github.com/paralus/paralus). Alternatively, we’ve setup a one-click installer on [Digital Ocean Marketpace](https://marketplace.digitalocean.com/apps/paralus) to make it even easier. We also have several quickstart guides on the the [blog](https://paralus.io/blog/).
+
+### 4. Is there support or an enterprise license available?
+
+Support for Paralus is available via [community Slack](https://join.slack.com/t/paralus/shared_invite/zt-1a9x6y729-ySmAq~I3tjclEG7nDoXB0A). Enterprise licensing and support is offered with Rafay’s Kubernetes Operations Platform.
+
+### 5. How is this different from Teleport?
+
+[Teleport](https://goteleport.com/) is an identity-aware access management and security for dynamic environments such as Kubernetes clusters, databases, and other cloud resources. Below are some of the salient features of Teleport:
+
+- Identity-Aware Access Management: Teleport provides fine-grained access control based on user identities, roles, and permissions, ensuring secure access to resources.
+- Single Sign-On (SSO): Users can log in once and access multiple resources without the need for repeated authentication, streamlining the user experience.
+- Multi-Factor Authentication (MFA): Enhances security by requiring additional authentication factors beyond just a password.
+- Session Recording and Audit Trails: Teleport records user sessions and access events, providing a comprehensive audit trail for compliance and security monitoring.
+
+While it also has the ability to provide secure access to Kubernetes clusters is a much more clunky solution as it:
+
+- requires setup per cluster
+- requires the use of Kubernetes secrets to mount the Teleport user token to the cluster
+- requires login first to a bastion Teleport server
+
+Paralus does not require any of these steps. It’s simply import your cluster and get started. User roles and responsibilities are not mixed with infrastructure deployments and management. Teleport does have some advanced functionality today such as session recordings which allow platform admins to capture the actions a user has performed and watch them again. These kinds of functionality are in the roadmap for Paralus in the near future.
+
+### 6. I installed Paralus correctly, however, I'm unable to access the password reset link?
+
+This happens when the password reset link has expired. The password reset link generated at the time of installing Paralus is valid only for 10 minutes. You can regenerate the URL by referring to our [password reset URL documentation](#password-reset-link-expired).
```

Comment:
@techmaharaj we need to update this, since now we have started generating password and this is not longer relavent. May be we can add applicable for paralus versions < v0.2.0

Reply:
Since v0.2.0, we can obtain default password using below and will be forced to reset during first login.

`kubectl logs -f --namespace default $(kubectl get pods --namespace default -l app.kubernetes.io/name='paralus' -o jsonpath='{ .items[0].metadata.name }') initialize | grep 'Org Admin default password:'`